MEMBER_TRAVEL_FUND.md

```diff
@@ -10,40 +10,72 @@ and the Node.js Foundation projects.
 
 ## Process
 
-Any member can apply for travel funds. Each request may take up to a month
-for the [Admin members](https://github.com/nodejs/admin#governance-and-current-members)
-to approve or decline.
+Any member can apply for travel funds.
 
 ### Request
-* Open a pull request against this repository which adds an entry to the table at the bottom of this file.
+
+* Open a pull request against this repository which adds an entry to the table
```

Comment:
Nit: I never really understood nor liked the "against this repository" phrasing - can we change that to something like "in this repository"?

Reply:
to?

Reply:
As said in the comment, something like "in this repository".

Reply:
Right; my suggestion is “to this repository”; “in this repository” doesn’t seem like the proper phrasing to me.

Reply:
Gotcha, sorry for the confusion.